src/schml/insert-casts.rkt

```diff
@@ -267,9 +280,64 @@
               ;; Justification: the runtime type is at
               ;; least as precise as t by semantics.
               (MBoxCastedSet! e1 e2 e2-ty))]
-         [else (error 'insert-casts/Mbox-set!T
+
+
+;;        <<<<<<< HEAD
```

Comment:
I am uncertain of the benefits of having a clean git history, but as these things come up feel free to teach me how to fix them. For instance, how would you "uncommit" this blob instead of fixing it with a new commit?